### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by georgehrke**
@tcitworld Should we also update our supported browsers?

- Firefox 45 is no longer supported by Mozilla. Latest ESR is 68, 60 ESR is still supported.
- Latest supported Chrome/Chromium is 77

Same for Edge. I think we should only support the Blink based version.

### CODE REVIEW

**Comment by tcitworld** [comment]
Just say the same as https://github.com/nextcloud/browserslist-config-nextcloud says ?
So basically https://browserl.ist/?q=last+2+versions%2C+ie+%3E%3D+11%2C+firefox+ESR%2Cnot+dead

File: README.md [hunk 1]
```diff
@@ -41,12 +40,21 @@ Check out our [FAQ](https://github.com/nextcloud/calendar/wiki/FAQs). If you don
...
+We’re also in [#nextcloud-calendar on freenode IRC](https://webchat.freenode.net/?channels=nextcloud-calendar).
 
-Testing is a great way to contribute without having to write source code.
-Although it's straight forward, setting up the development environment requires some knowledge and extra tools on your device.
+We’d like to thank [BrowserStack](https://www.browserstack.com) for providing us with a free subscription.
+
+## Nightly builds / Pre-releases
 
-We provide [nightly builds](https://nightly.portknox.net/calendar/?C=M;O=D) to enable everyone help testing without setting up the development environment.  
+Instead of setting everything up manually, you can just [download the nightly builds](https://nightly.portknox.net/calendar/?C=M;O=D) or [download a pre-release](https://github.com/nextcloud/calendar/releases) instead.
```
Comment (tcitworld) [inline README.md]: Nightly build link seems to be dead.

Comment (georgehrke) [reply README.md]: Yes, i know. Let me send an email to Steffen

**Comment by georgehrke** [comment]
> Just say the same as https://github.com/nextcloud/browserslist-config-nextcloud says ?
So basically https://browserl.ist/?q=last+2+versions%2C+ie+%3E%3D+11%2C+firefox+ESR%2Cnot+dead

I'm not a big fan of the current config, actually. It also includes Opera Mini and other weird esoteric browsers with next to no market share.

**Comment by tcitworld** [comment]
> Just say the same as https://github.com/nextcloud/browserslist-config-nextcloud says ?

Except maybe IE. 😬

**Comment by tcitworld** [comment]
> I'm not a big fan of the current config, actually. It also includes Opera Mini and other weird esoteric browsers with next to no market share.

So maybe closer to https://jamie.build/last-2-versions ?

**Comment by georgehrke** [comment]
But besides that, pretty much that list yes. Although I would still limit it to Firefox, Chrome, Edge, Safari and maybe opera. 

While our browser list config theoretically supports things like KaiOS browser, we realistically can't test / debug it.

**Comment by georgehrke** [comment]
I updated the browser list accordingly. Should we get this in? :)

File: README.md [hunk 2]
```diff
@@ -1,52 +1,60 @@
...
-* Edge 14+
-* Firefox 45+
+* Chrome/Chromium 76+
+* Edge 40+
+* Firefox 60+
 * Internet Explorer 11
-* Safari 10+
+* Safari 12.1+
 
 We don't support Internet Explorer 10 and below. Patches for IE9+ are accepted though.
```
Comment (tcitworld) [inline README.md]: Maybe just remove the second sentence.

Comment (georgehrke) [reply README.md]: Then I can also remove both sentences, since it should be obvious that we don't support IE 10 if the list says IE11

**Pull request merged**